tensorflow/java/src/gen/resources/tftypes.csv

```diff
@@ -0,0 +1,21 @@
+TF type,datatype code,Java type,Java boxed type,Creator?,default,Description
+Float,1,float,Float,y,0f,32-bit single precision floating point
+Double,2,double,Double,y,0,64-bit double precision floating point
+Int32,3,int,Integer,y,0,32-bit signed integer
+UInt8,4,byte,Byte,n,,8-bit unsigned integer
+Int16,5,,Short,n,(short)0,16-bit signed integer
+Int8,6,,Byte,n,(byte)0,8-bit signed integer
+String,7,byte,Byte,n,,An arbitrary sequence of bytes
+Complex64,8,,,n,,single-precision complex
+Int64,9,long,Long,y,0L,64-bit signed integer

```

Comment:
Simpifying the CSV is appealing, but In general they will be different types. In particular, some types like String do not have a primitive type equivalent. I guess I'd also like to avoid making semantics associations via string hacking.